tamr_toolbox/utils/version.py

```diff
@@ -49,8 +53,128 @@ def _as_float(version: str) -> float:
     return (version_split[0] * 1000) + version_split[1] + (version_split[2] / 10)
 
 
+def _get_tamr_versions_from_function_args(*args, **kwargs) -> List[str]:
```

Comment:
I agree it is complex, but it's a private function (_function) that is not designed to be user facing.  Sphinx will not even render the docstring by default.

(and indeed it can't even really be directly called - it takes any number of inputs and tells you the Tamr version of anything you put in, ignoring any non Tamr inputs)